bin/jupyter-dashboard-admin

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env node
+/**
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
+/**
+ * Command-line script which allows clearing of the Dashboard Server cache.
+ */
+
+var fs = require('fs');
+var hjson = require('hjson');
+var nconf = require('nconf');
+var path = require('path');
+var request = require('request');
+var urljoin = require('url-join');
+var yargs = require('yargs');
+
+// Config defaults are in an HJSON file in the root of the source tree
+var defaultConfig = path.join(__dirname, '..', 'config.json');
+var config = nconf.file({ file: defaultConfig, format: hjson });
+
+var DEFAULT_HOST = 'http://localhost:3000';
+
+// common options shared by all commands
+var commonOpts = {
+    'host': {
+        describe: 'dashboard server host and port',
+        default: DEFAULT_HOST,
+        type: 'string'
+    },
+    'auth-token': {
+        describe: 'authorization token for admin privelages, defaults to reading from config file',
+        type: 'string'
+    }
+};
+
+// command to clear the dashboard server cache
+var clear_cache_command = {
+    command: 'clear-cache [options]',
```

Comment:
The common options don't appear when passing `--help` for me so it's not clear how to pass the host or token without cracking the source. Can we fix this?